Bio/Data/PDBData.py

```diff
@@ -290,3 +290,99 @@
     "45A": "A", "4U3": "C", "8AG": "G", "8AN": "A", "8BA": "A", "8FG": "G",
     "8MG": "G", "8OG": "G", "8PY": "G", "8AA": "G", "85Y": "U", "8OS": "G",
 }
+
+# Solvent accesibility scales
+residue_max_acc = {
```

Comment:
You can give it an alias `residue_sasa_scales` (f ex) and do `residue_max_acc = residue_sasa_scales` in the DSSP module, to maintain compabitility there.